azurerm/helpers/azure/hdinsight.go

```diff
@@ -156,6 +156,34 @@ func SchemaHDInsightsExternalMetastore() *schema.Schema {
 	}
 }
 
+func SchemaHDInsightsMonitor() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"log_analytics_workspace_id": {
+					Type:     schema.TypeString,
+					Required: true,
+					ForceNew: false,
```

Comment:
Not sure sure I understood. It configured as `ForceNew: false,`. In *_updateMonitor tests I'm testing changing workspace_id and doesn't recreate cluster

Reply:
Sorry - Just saw `ForceNew`, not the `false`.  We only specify this property when setting it to `true` as it defaults to `false`, so it can be removed here.